content/courses/offline-transactions/durable-nonces.md

```diff
@@ -470,43 +470,41 @@ It will:
 
 Paste the following somewhere above the `describe` block.
 
-```ts
-async function createNonceAccount(
-  connection: Connection,
-  payer: Keypair,
-  nonceKeypair: Keypair,
-  authority: PublicKey,
-) {
-  // 2. Assemble and submit a transaction that will:
-  const tx = new Transaction().add(
-    // 2.1. Allocate the account that will be the nonce account.
-    SystemProgram.createAccount({
-      fromPubkey: payer.publicKey,
-      newAccountPubkey: nonceKeypair.publicKey,
-      lamports: 0.0015 * LAMPORTS_PER_SOL,
-      space: NONCE_ACCOUNT_LENGTH,
-      programId: SystemProgram.programId,
-    }),
-    // 2.2. Initialize the nonce account using the `SystemProgram.nonceInitialize` instruction.
-    SystemProgram.nonceInitialize({
-      noncePubkey: nonceKeypair.publicKey,
-      authorizedPubkey: authority,
-    }),
-  );
-
-  const sig = await sendAndConfirmTransaction(connection, tx, [
-    payer,
-    nonceKeypair,
-  ]);
-  console.log(
-    "Creating Nonce TX:",
-    `https://explorer.solana.com/tx/${sig}?cluster=custom&customUrl=http%3A%2F%2Flocalhost%3A8899`,
-  );
-
-  // 3. Fetch the nonce account.
-  const accountInfo = await connection.getAccountInfo(nonceKeypair.publicKey);
-  // 4. Serialize the nonce account data and return it.
-  return NonceAccount.fromAccountData(accountInfo!.data);
+```typescript
+const NONCE_ACCOUNT_LAMPORTS: number = 0.0015 * LAMPORTS_PER_SOL;
```

Comment:
Why do we need 0.0015 SOL for this? 

Could we use `getMinimumRentExemptBalance()` (or whatever it is called) for this?